.github/workflows/wheels.yml

```diff
@@ -136,16 +154,24 @@ jobs:
           # A shallow clone can work when the following issue is resolved:
           # https://github.com/actions/checkout/issues/338
           fetch-depth: 0
-      # Used to push the built wheels
-      - uses: actions/setup-python@v3
+
+      # Used to push the built sdist
+      - uses: conda-incubator/setup-miniconda@v2
         with:
-          # Build sdist on lowest supported Python
+          auto-update-conda: true
+          # Really doesn't matter what version we upload with
+          # just the version we test with
           python-version: '3.8'
+          channels: conda-forge,defaults

```

Comment:
I had bad experiences in the past with the defaults channel (when mixed with conda-forge), and I've been using conda-forge only. If you've got a reason to have defaults surely fine, but otherwise I'd probably remove it.